tools/automator/automator.sh

```diff
@@ -287,7 +295,7 @@ work() { (
 
     git add --all
 
-    if ! git diff --cached --quiet --exit-code; then
+    if ! git diff --cached --quiet --exit-code "${git_exclude[@]}"; then
```

Comment:
isn't this including the file, not excluding it? I may be understanding wrong though

Reply:
The exclude happens because I prefixed the string with `:^` above. Maybe I didn't explain that enough in the code.